src/main/java/TicTacToe/Session.java

```diff
@@ -0,0 +1,41 @@
+package TicTacToe;
+
+class Session {
```

Comment:
What's the intent behind a `Session`? Right now it feels like its only responsibility is to change the player's symbols _after_ they were given default ones already.
I wonder if that's enough to necessitate a separate class?